nixos/tests/morty.nix

```diff
@@ -0,0 +1,33 @@
+import ./make-test.nix ({ pkgs, ... }:
+
+{
+  name = "morty";
+  meta = with pkgs.stdenv.lib.maintainers; {
+    maintainers = [ leenaars ];
+  };
+
+  nodes =
+    { mortyProxyWithKey =
+
+      { config, pkgs, ... }:
+      { services.morty = {
+          enable = true;
+	  key = "78a9cd0cfee20c672f78427efb2a2a96036027f0";
+	  port = 3001;
+	};
```

Comment:
Please unindent `enable` by 2 spaces, then indent the rest to be in line with the enable line.